### ISSUE TITLE
Release binary permissions problems (macOS)

### ISSUE THREAD

**Issue by genericptr** [OmniSharp/omnisharp-roslyn#1817]
I'm on macOS Catalina and not able to execute the language server. First thing I notice is that the permissions of the included files seem to be incorrect so I can't execute the binary but doing a chmod +x causes other errors so I'm not sure what the problem is.

omnisharp-osx$ ./run
-bash: ./run: Permission denied

![Screen Shot 2020-06-02 at 10 34 23 AM](https://user-images.githubusercontent.com/10937186/83477107-a5c24280-a4bc-11ea-874b-d2105b92c7c4.png)

**Comment by lxbarbosa** [comment]
I confirm, `run` has only read-write permissions!

**Comment by genericptr** [comment]
I just changed the permissions with chmod +x now and it seems to be working! I upgraded some .NET related stuff yesterday though so something may have changed. Sorry I can't track down the problem so I guess the issue now is the permissions are incorrect so I'll close the issue.

**Comment by lxbarbosa** [comment]
@genericptr 

It still an issue as most users will eventually realize that too.

**Comment by genericptr** [comment]
You're right. I'll let the actual moderators decide what to do. I can't reproduce the first set of errors but I don't know if they were related to the permissions or not.

**Comment by lxbarbosa** [comment]
@genericptr 

It might be because Linux binaries files are all set with execute permissions!

**Comment by genericptr** [comment]
I just downloaded 1.38.0 and get the same error! Really should fix this....
